build.gradle

```diff
@@ -41,9 +41,10 @@ dependencies {
     annotationProcessor "org.projectlombok:lombok:${lombokVersion}"
     compileOnly "org.projectlombok:lombok:${lombokVersion}"
     implementation "com.connexta.search:index-api-rest-spring-stubs:${indexApiVersion}"
-    implementation "com.connexta.search:query-api-rest-spring-stubs:0.2.0"
+    implementation "com.connexta.search:query-api-rest-spring-stubs:0.3.0"
     implementation "io.springfox:springfox-swagger2:${springfoxSwaggerVersion}"
     implementation "io.springfox:springfox-swagger-ui:${springfoxSwaggerVersion}"
+    implementation "org.springframework:spring-web:5.2.0.RELEASE"
```

Comment:
❓Why was this added?

Reply:
I was able to full build without this. If it is needed, it should be caught in the hero.